src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -487,36 +487,39 @@ export default class GroupComparisonStore {
         invoke:()=>Promise.resolve(pickProteinEnrichmentProfiles(this.molecularProfilesInActiveStudies.result!))
     });
 
-    @observable.ref private _mutationEnrichmentProfile:MolecularProfile|undefined = undefined;
-    readonly mutationEnrichmentProfile = remoteData({
+    @observable.ref private _studyMutationEnrichmentProfileSet:{[id:string]:MolecularProfile} = {};
```

Comment:
I think a "Set" is by definition an array of things, wherease, this is a map, no?